guides/hack/20-attributes/07-predefined-attributes.md

```diff
@@ -206,6 +208,17 @@ mock class can exist.
 
 Mock classes *cannot* extend types `vec`, `dict`, and `keyset`, or the Hack legacy types `Vector`, `Map`, and `Set`.
 
+## __Newable
+
+This attribute is used to annotate reified type parameters to ensure that they are only instantiated with classes on which `new` can be safely called.  A common pattern is:
+```Hack
+function f<<<__Newable>> reify T as A>(): T {
+  return new T();
+}
+```
+where the class `A` is either final or annotated with `__ConstantConstructor`.  As a result the function `f` can only be applied to a _non-abstract_ class `C`.  The `as A` constraint ensures that the interface of the constructor of `C` is uniquely determined by the interface of the constructor of class `A`. 
```

Comment:
I believe you meant `__ConsistentConstruct` and not `__ContantConstructor`.

Reply:
Yes, thanks.  Will fix in a second.

Reply:
`Construct`, not `Constructor`

Reply:
Silly me.  Should have double checked the exact keyword, thanks.